gcc/rust/resolve/rust-ast-resolve-expr.cc

```diff
@@ -173,7 +173,8 @@ ResolveExpr::visit (AST::IdentifierExpr &expr)
     }
   else
     {
-      rust_error_at (expr.get_locus (), "failed to find name: %s",
+      rust_error_at (expr.get_locus (), ErrorCode ("E0425"),
+		     "cannot find value `%s` in this scope",
```

Comment:
For this format string you will need to use "cannot find value %<%s%> in this scope" using %< inserts the quotes for us in a portable for gcc

Reply:
`%qs` would also be simpler and achieve the same thing if I'm not mistaken

Reply:
fixed :)